File: Anagrams/makeAnagram.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int makeAnagram(string a, string b){
    vector<int> freq(26);
    int count = 0;
    //range-based for loop is a C++11 extension
    for(char c : a) {++freq[c - 'a'];}
    for(char c : b) {--freq[c - 'a'];}
    for(int val : freq) {count += abs(val);}
    return count;
}

int main(){
    string a;
    getline(cin, a);
    string b;
    getline(cin, b);

    int count = makeAnagram(a, b);
    cout<<count<<endl;
    return 0;
}

File: Anagrams/makeAnagram.py

def makeAnagram(a, b):
    seq = [0]*26
    count = 0
    for item in a:
        seq[ord(item) - ord('a')] += 1
    for item in b:
        seq[ord(item) - ord('a')] -= 1
    for item in seq:
        count += abs(item)
    return count

if __name__ == '__main__':
    a = input()
    b = input()
    res = makeAnagram(a, b)
    print(res)
    